autoPyTorch/api/base_task.py

```diff
@@ -616,7 +626,8 @@ def _do_traditional_prediction(self, num_run: int, time_for_traditional: int) ->
                         "Traditional prediction for %s failed with run state %s and additional output: %s.",
                         classifier, str(status), str(additional_info),
                     )
-        return num_run
+        self._logger.debug("Run history traditional: {}".format(run_history))
```

Comment:
Can you write this to a json file in `.autopytorch` directory so we have this for debug?

Reply:
sure